hackathon_site/dashboard/frontend/src/components/cart/CartCard/CartCard.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import styles from "./CartCard.module.scss";
+import Typography from "@material-ui/core/Typography";
+import Grid from "@material-ui/core/Grid";
+import {
+    Card,
+    CardMedia,
+    Select,
+    FormControl,
+    MenuItem,
+    IconButton,
+    Paper,
+    InputLabel,
+} from "@material-ui/core";
+import CloseIcon from "@material-ui/icons/Close";
+
+const Selections = (currentStock) => {
+    const items = [];
+    let i = 0;
+
+    for (i = 1; i < currentStock + 1; i++) {
+        items.push(
+            <MenuItem key={i} role="quantity" value={i.toString()}>
+                {i}
+            </MenuItem>
+        );
+    }
+    return items;
+};
+
+export const CartCard = ({
+    image,
+    title,
+    currentStock,
+    closeChange,
+    handleChange,
+    item,
+}) => {
+    return (
+        <Grid className={styles.Cart} item>
+            <Paper className={styles.Cart} elevation={0}>
+                <CardMedia
+                    className={styles.CartPic}
+                    image={image}
+                    alt={title}
+                    component="img"
+                />
+            </Paper>
+            <Card className={styles.CartParts} elevation={0}>
+                <Typography variant="body2" className={styles.CartName}>
+                    {" "}
+                    {title}{" "}
+                </Typography>
+                <FormControl
+                    variant="outlined"
+                    size="small"
+                    className={styles.CartFormControl}
+                >
+                    <InputLabel shrink={true} id="Qty">
+                        {" "}
+                        Qty{" "}
+                    </InputLabel>
+                    <Select
+                        labelId="Qty"
+                        value={item}
+                        onChange={handleChange}
+                        role="selecter"
+                        name="quantity"
+                    >
+                        {Selections(currentStock)}
+                    </Select>
+                </FormControl>
+            </Card>
+            <IconButton
+                size="small"
+                className={styles.CartClose}
+                onClick={closeChange}
+                role="close"
+            >
+                <CloseIcon className={styles.CartClose} />
```

Comment:
IconButton and CloseIcon have the same class which doesn't make sense. Make 2 separate classes.